src/connector/null-connector.ts

```diff
@@ -20,49 +20,49 @@ export class NullConnector extends Connector<NullChannel, NullPrivateChannel, Nu
     /**
      * Listen for an event on a channel instance.
      */
-    listen(name: string, event: string, callback: Function): NullChannel {
+    listen(_name: string, _event: string, _callback: CallableFunction): NullChannel {
         return new NullChannel();
     }
 
     /**
      * Get a channel instance by name.
      */
-    channel(name: string): NullChannel {
+    channel(_name: string): NullChannel {
```

Comment:
Why do these need to be renamed?

Reply:
This tells ESlint that it's an unused argument